yt/yt/ytlib/chunk_client/input_chunk_slice.cpp

```diff
@@ -464,7 +464,7 @@ TInputChunkSlice::TInputChunkSlice(
 
     if (protoChunkSlice.has_row_count_override() || protoChunkSlice.has_data_weight_override()) {
         YT_VERIFY(protoChunkSlice.has_row_count_override() && protoChunkSlice.has_data_weight_override());
-        OverrideSize(protoChunkSlice.row_count_override(), std::max<i64>(1, protoChunkSlice.data_weight_override() * chunkSlice.GetInputChunk()->GetColumnSelectivityFactor()));
+        OverrideSize(protoChunkSlice.row_count_override(), std::max<i64>(1, protoChunkSlice.data_weight_override() * chunkSlice.GetInputChunk()->GetSelectivityFactor()));
```

Comment:
I think using both factors here might break [this](https://github.com/ytsaurus/ytsaurus/blob/main/yt/yt/ytlib/table_client/chunk_slice_fetcher.cpp#L326) usage within `TChunkSliceFetcher`. 
The chunk slice fetcher computes the override data weight as `slice_row_count * chunk_data_weight_per_row`, which already approximates the real number of rows inside the slice.

Reply:
Disabled chunk slice statistics for operations and tables that use `TChunkSliceFetcher`.